lib/assets/javascripts/chapman_stories_feed.js

```diff
@@ -0,0 +1,191 @@
+
+  "use strict";
+
+  var pluginName = 'chapmanStoriesFeed';
+
+  // undefined is used here as the undefined global variable in ECMAScript 3 is
+  // mutable (ie. it can be changed by someone else). undefined isn't really being
+  // passed in so we can ensure the value of it is truly undefined. In ES5, undefined
+  // can no longer be modified.
+
+  // window and document are passed through as local variables rather than global
+  // as this (slightly) quickens the resolution process and can be more efficiently
+  // minified (especially when both are regularly referenced in your plugin).
+
+  // Create the defaults once
+  var defaults = {
+    baseUrl: 'http://social.localchapman.edu:3000/',
+    storiesPerPage: 4,
+
+    // The values are expected to be data attributes on the element. The user supplies them
+    // via the Cascade interface.
+    dataTitle: 'Chapman Stories Feed',
+    dataBgColor: null,
+    dataButtonLabel: 'View more stories',
+    dataButtonUrl: 'https://blogs.chapman.edu/',
+    dataFeedUrl: 'https://blogs.chapman.edu/',
+    dataTopStoryUrl: 'https://blogs.chapman.edu/'
+  };
+
+  // The actual plugin constructor
+  function Plugin(element, options) {
+    this.element = element;
+    this.$element = $(element);
+
+    // jQuery has an extend method which merges the contents of two or
+    // more objects, storing the result in the first object. The first object
+    // is generally empty as we don't want to alter the default options for
+    // future instances of the plugin
+    this.settings = $.extend({}, defaults, options);
+    this._defaults = defaults;
+    this._name = pluginName;
+    this.init();
+  }
+
+  // Avoid Plugin.prototype conflicts
+  $.extend(Plugin.prototype, {
+    init: function() {
+      // Place initialization logic here. You already have access to the DOM element and
+      // the options via the instance, e.g. this.element and this.settings.
+
+      // Data attribute values
+      this.feedUrl          = this.parseFeedUrl();
+      this.topStoryUrl      = this.$element.data('chapmanStoriesTopStoryUrl');
+      this.title            = this.$element.data('chapmanStoriesTitle') || this.settings.dataTitle;
+      this.backgroundColor  = this.$element.data('chapmanStoriesBgColor') || this.settings.dataBgColor;
+      this.buttonLabel      = this.$element.data('chapmanStoriesButtonLabel') || this.settings.dataButtonLabel;
+      this.buttonUrl        = this.$element.data('chapmanStoriesButtonUrl') || this.settings.dataButtonUrl;
+      this.buildOutWidget();
+    },
+
+    //this is a temporary fix to remove trailing slash from blog site urls(that is how they are stored on inside db)
+    //Ideally, inside query should be able to match urls with or without trailing slash
+    parseFeedUrl: function() {
+      var parsedUrl = this.$element.data('chapmanStoriesFeedUrl') || this.settings.dataFeedUrl;
+      parsedUrl = parsedUrl.replace(/\/?$/, '');
+      return parsedUrl;
+    },
```

Comment:
This isn't used, is it? I think this is a vestige of previous version. If so, please remove.